app/assets/stylesheets/landing.scss

```diff
@@ -0,0 +1,13 @@
+.home-back-image{
+  color: rgba(255,255,255,0.6);
+  background: linear-gradient(rgba(0,0,0,.5), rgba(0,0,0,.5)),url('https://www.kenresearch.com/blog/wp-content/uploads/2017/11/Jordan-telecommunications-market.jpg');

```

Comment:
download pic, at least for speed and in case it will be not available

Reply:
of course download. ALL resources hosted locally in your app.